test_tab.py

```diff
@@ -34,7 +40,7 @@
 from ac_classes import BiDataModel as bdmodel
 from ac_classes import MultiDataModel as mdmodel
 from nonlinear_regression import NonLinearRegression as nlr
-from anova import main as anova_main
+# from anova import main as anova_main
```

Comment:
Please un-comment this line 43

Reply:
We still use anova_main [here](https://github.com/AnalyticsCal/AnalyticsCal-Classroom/blob/d54333b1a52a50fc5a1c2a59a694d58dd6178034/test_tab.py#L819)